components/combobox/combobox.jsx

```diff
@@ -382,6 +412,69 @@ class Combobox extends React.Component {
 		}
 	}
 
+	getCustomPopoverProps = (body, { assistiveText }) => {
+		/*
+		 * Generate the popover props based on passed in popover props. Using the default behavior if not provided by passed in popover
+		 */
+		const popoverBody = (
+			<div>
+				<div className="slds-assistive-text" id={`${this.getId()}-label`}>
+					{assistiveText.popoverLabel}
+				</div>
+				{body}
+			</div>
+		);
+
+		const popoverFooter = (
+			<div>
+				<button
+					className="slds-button slds-button_neutral"
+					id="cancel-button"
+					onClick={(e) => {
+						// onClose(e, { trigger: 'cancel' });
+						this.handleClose(e, { trigger: 'cancel' });
+					}}
+				>
+					Cancel
+				</button>
+				<button
+					className="slds-button slds-button_brand"
+					ref={this.lastTab}
+					id="done-button"
+					onClick={this.handleClose}
+				>
+					Done
+				</button>
+			</div>
+		);
+
+		const defaultPopoverProps = {
+			ariaLabelledby: `${this.getId()}-label`,
+			align: 'bottom',
+			body: popoverBody,
+			className: 'slds-popover_full-width',
+			footer: popoverFooter,
+			footerClassName: 'slds-popover__footer_form',
+			hasNoNubbin: true,
+			id: `${this.getId()}`,
+			isOpen: this.state.isOpen,
+			hasNoTriggerStyles: true,
+			onOpen: this.handleOpen,
+			onClose: this.handleClose,
+			onRequestClose: this.handleClose,
+		};
+
+		/* Mixin passed popover's props if there is any to override the default popover props */
```

Comment:
Please use a different word than `mixin`, since that has another meaning in React.

Reply:
What is a better word? I copied that from the filter component.

Reply:
I probably wrote it then. 😊 

Umm, merge in would be better, I think

Reply:
Haha ok.